lib/attributor/types/tempfile.rb

```diff
@@ -31,6 +31,9 @@ def self.load(value,context=Attributor::DEFAULT_ROOT_CONTEXT, **options)
       super
     end
 
+    def self.family
+      String.family
```

Comment:
ya, same here...from the point of view of an example, that will be a string...but from a point of view of the object itself, it's different...grr